@AH-64D Apache Longbow/Addons/fza_ah64_systems/functions/drivetrain/fn_drivetrainNoseGearbox1.sqf

```diff
@@ -0,0 +1,19 @@
+/* ----------------------------------------------------------------------------
+Function: fza_systems_fnc_drivetrainNoseGearbox1
```

Comment:
Why does this exist?

Reply:
Future growth, there's a plan...forgot about it, will implement after merge.